<?xml version="1.0"?>
<launch>
	<arg name="detector_GUI" value = "true"/>

<!-- ================================================================== -->
	<node pkg="camera" type="video" name="video" output="screen">
		<param name="videoport" value="0"/>
		<param name="hz" value="10"/>
	</node>

	<node pkg="detecter" type="circle_node" name="circle_node">
        <param name="/detecter/circle_resize_flag" value="1"/>
        <param name="/detecter/circle_resize_width" value="320"/>
        <param name="/detecter/circle_resize_height" value="240"/>
        <param name="/detecter/circle_show_image_result_flag" value="1"/>
	</node>

	<group if="$(arg detector_GUI)">
    	<node pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find detecter)/rviz/circle.rviz"/>
    </group>
</launch>
